Automattic/wp-calypso
# Adds a static list of DIFM categorized themes

#### Changes proposed in this Pull Request

* Adds a static list of themes to be passed to the DesignPicker.
(This builds on top of https://github.com/Automattic/wp-calypso/pull/57185)

#### Testing instructions



* Start the flow at `/start/do-it-for-me?flags=signup/design-picker-categories`.
* There should be 4 categories and the design picking step should look like this:
![image](https://user-images.githubusercontent.com/5436027/138863309-8b20c611-58fb-43dd-bfb1-7fda963574ec.png)
